SherpaFrontEnd.Tests/Shared/NavMenuTest.cs

```diff
@@ -15,4 +15,26 @@ public void Should_have_an_anchor_with_templates_ref()
         
         Assert.NotNull(element);
     }
+    
+    [Fact]
+    public void ShouldHaveAnAnchorWithTeamsPageRef()
+    {
+        var ctx = new TestContext();
+        var component = ctx.RenderComponent<NavMenu>();
+
+        var element = component.Find("a[href='teams-list-page']");
+
+        Assert.NotNull(element);
+    }
+
+    [Fact]
+    public void ShouldHaveTheTitleTeams()
+    {
+        var ctx = new TestContext();
+        var component = ctx.RenderComponent<NavMenu>();
+
+        var element = component.Find("a[href='teams-list-page']");
+
+        Assert.Contains("Teams", element.InnerHtml);
+    }
```

Comment:
Why don't we focus this two tests in the user experience like this.

1. Look for a "a" element with the text "Teams"
2. Verify that the href property is the one that you want

The end user will look for the "Teams" word

Reply:
Sure then, I will re-write it the test